stablehlo/tests/CheckOps.td

```diff
@@ -81,6 +81,42 @@ def CHECK_ExpectAlmostEqOp :
   let assemblyFormat = "$lhs `,` $rhs attr-dict `:` type($lhs)";
 }
 
+def CHECK_ExpectIsCloseOp :
+    Op<CHECK_Dialect, "expect_is_close", [AllTypesMatch<["lhs", "rhs", "input"]>]> {
+  let summary = [{Checks that the tensor operands are equal within a tolerance tensor}];
+  let description = [{
+    Verifies that the tensor operands with floating-point element
+    type are equal within an user-defined tolerance tensor:
+
+      abs(lhs - rhs) < abs_error
+
+    abs_error is typically
+
+      atol + rtol * abs(rhs)
+
+    where atol must be chosen carefully to match rhs magnitude.
+
+    When rhs operand is non-finite, expect_allclose is equivalent to
+    expect_eq with respect to that operand.
+
+    Input argument is used only when reporting mismatches between lhs
```

Comment:
What does this signify? Is this some extra information that `lhs`, `rhs` cannot provode?

Reply:
See https://github.com/openxla/stablehlo/pull/2357#discussion_r1608025497 .
Indeed, `lhs` and `rhs` is not sufficient to determine the cause of the test failure. There is `index` but that is inconvenient to use (see the discussion above). `input` is the most useful parameter that determines the cause why `lsh` and `rhs` do not match.